docs/package.json

```diff
@@ -3,13 +3,14 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "start-storybook -p 9002 -s assets",
-    "build": "NODE_ENV=production build-storybook -o public -s assets"
+    "start": "NODE_OPTIONS='--max-old-space-size=8192' start-storybook -p 9002 -s assets",
+    "build": "NODE_OPTIONS='--max-old-space-size=8192' NODE_ENV=production build-storybook -o public -s assets"
   },
   "dependencies": {
     "@babel/core": "7.22.8",
-    "@emotion/react": "^11.4.0",
-    "@emotion/styled": "^11.3.0",
+    "@commercetools-frontend/babel-preset-mc-app": "21.25.2",
```

Comment:
This had to be changed from a devDependency to a regular dependency. Otherwise babel compilation with `@emotion/babel-plugin` didn't work properly.